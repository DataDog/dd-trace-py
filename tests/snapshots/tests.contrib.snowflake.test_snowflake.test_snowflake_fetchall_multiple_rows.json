[[
  {
    "name": "sql.query",
    "service": "snowflake",
    "resource": "select a, b from t;",
    "trace_id": 0,
    "span_id": 1,
    "parent_id": 0,
    "type": "sql",
    "error": 0,
    "meta": {
      "_dd.p.dm": "-0",
      "component": "snowflake",
      "db.application": "PythonConnector",
      "db.name": "mock-db-name",
      "db.schema": "mock-schema-name",
      "db.user": "mock-user",
      "db.warehouse": "mock-warehouse-name",
      "language": "python",
      "out.host": "mock-account.snowflakecomputing.com",
      "runtime-id": "1658e263f3084eaca410818e944d7d7e",
      "sfqid": "mocksfqid-29244a72-90a0-4309-9168-9dcf67721f6d"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.measured": 1,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "db.row_count": 2,
      "network.destination.port": 443,
      "process_id": 88705
    },
    "duration": 3837000,
    "start": 1660069792128531000
  }]]
