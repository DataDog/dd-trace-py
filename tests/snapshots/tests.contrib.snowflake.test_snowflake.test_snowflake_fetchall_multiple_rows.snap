[[{"name" "snowflake.query"
   "service" "db"
   "resource" "select a, b from t;"
   "type" "sql"
   "error" 0
   "span_id" 0
   "trace_id" 0
   "parent_id" nil
   "start" 1633548715861747000
   "duration" 4090300
   "meta" {"runtime-id" "d55668de0f204c299502f460c9dbd13d"
           "out.host" "mock-account.snowflakecomputing.com"
           "db.name" "mock-db-name"
           "db.user" "mock-user"
           "db.application" "PythonConnector"
           "db.schema" "mock-schema-name"
           "db.warehouse" "mock-warehouse-name"}
   "metrics" {"_dd.agent_psr" 1.0
              "_dd.measured" 1
              "_sampling_priority_v1" 1
              "system.pid" 5179
              "out.port" 443
              "db.rowcount" 2
              "sql.rows" 2
              "_dd.tracer_kr" 1.0}}]]
